react/components/Buttons/AddToCart.js

```diff
@@ -32,13 +32,15 @@ class AddToCart extends Component {
     return (
       <div className={`actions--add-to-cart tc pa5 ${isModalOpen ? 'w-100 left-0 bg-white z-4 bottom-0 bt b--light-gray' : ''}`}>
         <Button type="submit" onClick={onSubmit} isLoading={isLoading} block>
-          <FormattedMessage id="product-customizer.add-to-cart" />
-          <ProductPrice
-            showLabels={false}
-            showListPrice={false}
-            sellingPrice={total}
-            listPrice={total}
-          />
+          <div className="flex flex-grow-1 justify-between items-center">
```

Comment:
`flex-grow` is normally a child property of a parent with `flex`. Does it make sense here?

Reply:
I thought the parent was `flex` too, but I was wrong. Setting it to `w-100`